nmkapp/templates/request_password.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% load nmk_tags %}
+
+{% block content %}
+
+<h1 style="text-align: center;">{% trans "SHARKZ.BET - ADD PASSWORD" %}</h1>
+{% if nonvalid %}
+{% trans "Invalid request to add password" %}
+{% else %}
+    {% if reset %}
+    {% trans "Password added successfully" %}
+    <br/>
+    <br/>
+    <div>
+        <a href="/login">{% trans "Back to login" %}</a>
+    </div>
+    {% else %}
+    <form method="post" action="/requestpassword">
+        {% csrf_token %}
+
+        <fieldset>
+            <div>{% blocktrans %}Adding password for user <b>{{ email }}</b>{% endblocktrans %}</div>
```

Comment:
razmisljam naglas...ipak je ovo malo "nekonvencionalni" pristup, mozda nam treba neki paragraf ovde da objasnimo zasto trazimo password, i da to nije google password koji trazimo! (samo kazem, ne treba da se ispravlja u ovom commitu!)